sus/collections/array.h

```diff
@@ -405,6 +427,115 @@ class Array final {
         sus::iter::IterRefCounter::empty_for_view(), storage_.data_, N);
   }
 
+  /// Compares two Arrays.
+  ///
+  /// Satisfies sus::cmp::StrongOrd<Array<T, N>> if sus::cmp::StrongOrd<T>.
+  ///
+  /// Satisfies sus::cmp::Ord<Array<T, N>> if sus::cmp::Ord<T>.
+  ///
+  /// Satisfies sus::cmp::PartialOrd<Array<T, N>> if sus::cmp::PartialOrd<T>.
+  /// #[doc.overloads=array.cmp.array]
+  template <class U>
+    requires(::sus::cmp::ExclusiveStrongOrd<T, U>)
+  constexpr friend std::strong_ordering operator<=>(
```

Comment:
can we write `friend constexpr` as the order for these keywords consistently (the one above is in that order)?